app/controllers/api_resource_evaluations_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ApiResourceEvaluationsController < ApiApplicationController
+  def update
+    evaluation = params['evaluation']
```

Comment:
It should be verified that evaluation is not nil before saving.
I found that if the 'evaluation' parameter is not present in the body (because misspelled or body is empty), then` `evaluation=nil` and this is saved to the database, which is undesirable.